Gordon360/Services/NewsService.cs

```diff
@@ -70,7 +70,15 @@ orderby snc.SortOrder
 
     public IEnumerable<StudentNewsCategoryViewModel> GetNewsCategories()
     {
-        return context.StudentNewsCategory.OrderBy(c => c.SortOrder).Select<StudentNewsCategory, StudentNewsCategoryViewModel>(c => c);
+        var excludedCategoryIds = new[] { 2, 3 };
```

Comment:
This should probably be a constant, rather than being newly allocated every time. Also, since C# version 12 (with .NET 8), the better way to declare a collection is using [collection expressions](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/collection-expressions), because the compiler chooses the best way to create the collection.
```suggestion
        const int[] ExcludedCategoryIds = [ 2, 3 ];
```